src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const tbody = table.querySelector('tbody');
+const rows = tbody.querySelectorAll('tr');
+const thead = document.querySelector('thead');
+const theadRow = thead.firstElementChild;
+const secondPositionTh = document.createElement('th');
+const secondPositionThFoot = document.createElement('th');
+
+secondPositionTh.textContent = 'Position';
+secondPositionThFoot.textContent = 'Position';
+theadRow.insertBefore(secondPositionTh, theadRow.lastChild.previousSibling);
+
+const tfoot = document.querySelector('tfoot');
+const tfootRow = tfoot.firstElementChild;
+
+tfootRow.insertBefore(secondPositionThFoot, tfootRow.lastChild.previousSibling);
+
+rows.forEach((row) => {
+  const positionCell = row.querySelector('td:nth-child(2)');
+  const newPositionCell = positionCell.cloneNode(true);
+
+  row.insertBefore(newPositionCell, row.lastChild.previousSibling);
+});
```

Comment:
You can simplify it like this
```suggestion
const table = document.querySelector('table');

for (const row of table.rows) {
  const column = row.cells[1].cloneNode(true);

  row.lastElementChild.before(column);
}
```

Reply:
already did this

Reply:
check my second commit